extras/rya.pcj.fluo/pcj.fluo.app/src/main/java/org/apache/rya/indexing/pcj/fluo/app/IncrementalUpdateConstants.java

```diff
@@ -39,7 +39,11 @@
     public static final String CONSTRUCT_PREFIX = "CONSTRUCT";
     public static final String PERIODIC_QUERY_PREFIX = "PERIODIC_QUERY";
     
+    //binding name reserved for periodic bin id for periodic query results
     public static final String PERIODIC_BIN_ID = PeriodicQueryResultStorage.PeriodicBinId;
+    
+    //constant used for storing Kafka connection information in Fluo
+    public static final String KAFKA = "KAFKA";
```

Comment:
Might as well turn those comments into javadocs.

Reply:
Realized I wasn't actually using this.  Deleted.